chatgpt_wrapper/backends/openai/repl.py

```diff
@@ -165,7 +165,6 @@ async def do_user_register(self, username=None):
     async def check_login(self):
         user_count = self.session.query(User).count()
         if user_count == 0:
-            self.console.print("No users in database. Creating one...", style="bold red")
```

Comment:
This is incorrect, we want to keep the message. Instead, replace this with the proper call to `print_status_message()` from `core.util`

Reply:
Done!